h5p-bildetema-words-grid-view/src/components/Word/Word.module.scss

```diff
@@ -1,12 +1,20 @@
 @use "../../../../common/abstracts" as *;
 
 .word {
+  position: relative;
```

Comment:
Already exists on line 8

Reply:
A tip is to use alphabetical order when adding styles, so it is easier to find each styling.